lib/index.js

```diff
@@ -10,7 +10,7 @@
         date = +date || date;
```

Comment:
How about a check if the date string is empty before the integer conversion:
```
if (date.length === 0) return chunk.write('');
```

Should work with the date zero.